communication/slack-config/usergroups.yaml

```
@@ -290,3 +290,13 @@ usergroups:
       - mrbobbytables
       - nikhita
       - palnabarun
+
+  - name: kubestellar-devs
+    long_name: KubeStellar Development Team
+    description: Active developers of KubeStellar on #kubestellar-dev
```

```suggestion
    description: "Active developers of KubeStellar on #kubestellar-dev"
```
Since `#kubestellar-dev` will be considered as a comment and not actually displayed when viewed on slack.
(probably needs to be fixed for the `kcp-devs` handle too)